packages/terra-form-select/src/single/Frame.jsx

```diff
@@ -317,6 +317,7 @@ class Frame extends React.Component {
       event.preventDefault();
       this.openDropdown();
     } else if (keyCode === KeyCode.KEY_ESCAPE) {
+      this.select.focus();
```

Comment:
was it happening only on single select...???

Reply:
Yes